lib/node_modules/@stdlib/utils/some-own-by/README.md

```diff
@@ -0,0 +1,238 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# someOwnBy
+
+> Test whether an object contains at least `n` own properties which pass a test implemented by a predicate function.
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var someOwnBy = require( '@stdlib/utils/some-own-by' );
+```
+
+#### someOwnBy( obj, n, predicate\[, thisArg ] )
+
+Tests whether an obj contains at least n own properties which pass a test implemented by a predicate function.
+
+```javascript
+function isNegative( value ) {
+    return ( value < 0 );
+}
+
+var obj = {
+    'a': 1,
+    'b': -2,
+    'c': 3,
+    'd': -1
+};
+
+var bool = someOwnBy( obj, 2, isNegative );
+// returns true
+```
+
+Once the function finds n successful properties, the function immediately returns true.
+
+```javascript
+function isPositive( value ) {
+    if ( value < 0 ) {
+        throw new Error( 'should never reach this line' );
+    }
+    return ( value > 0 );
+}
+
+var obj = {
+    'a': 1,
+    'b': 2,
+    'c': -3,
+    'd': 4
+};
+
+var bool = someOwnBy( obj, 2, isPositive );
+// returns true
+```
+
+The invoked function is provided three arguments:
+
+-   `value:` object property value
+-   `key`: object property key
+-   `obj`: input object
```

Comment:
```suggestion
-   `value`: object property value
-   `key`: object property key
-   `obj`: input object
```


Not sure if this is correct, please refer nearby packages.

Reply:
Yeah, you're correct! I just mistyped.